build-utils/reportdeploy.sh

```diff
@@ -51,4 +53,4 @@ OTHER_BRANCHES=$(other_branches_same_commit | xargs)
 AUTHOR=$(commit_author)
 # Passing to sheetappend the GCP credentials we are expecting to be piped into this script
 cat /dev/stdin |
-   sheetappend $RELEASE_SHEET_ID $DATE $TIME "$COMPONENT" $ENVIRONMENT $BRANCH "$TAG" $SHORT_GIT_SHA "$AUTHOR" $CIRCLE_BUILD_NUM "$CIRCLE_BUILD_URL" "$OTHER_BRANCHES"
+   sheetappend $RELEASE_SHEET_ID $DATE $TIME "$COMPONENT" $ENVIRONMENT $BRANCH "$TAG" $SHORT_GIT_SHA "$AUTHOR" "$CIRCLE_BUILD_HYPERLINK" "" "$OTHER_BRANCHES"
```

Comment:
Instead of having a column for the build number and another for the build url, we are just going to create a hyperlink in the spreadsheet. This allows us to see an important column that is currently hiding (build branches). Leaving the URL blank column in place for now.